### DISCUSSION TITLE
hi all,  Modify the source code then with which Docker file to build a new image,there are so many Docker file.

### DISCUSSION THREAD

**Discussion by shaojiezhao** [langflow-ai/langflow#1379] [General]
hi all,  Modify the source code then with which Docker file to build a new image,there are so many Docker file.

**Comment by Wellington-LMD** [comment]
Did you get it?

**Comment by yashugupta786** [comment]
@shaojiezhao @shaojiezhao  I have also modified source code not sure which docker file to use and compose file to run this . Any help
